Gruntfile.js

```diff
@@ -552,16 +552,17 @@ module.exports = function(grunt) {
     'create-windows-installer:custom',
   ]);
 
-  grunt.registerTask('linux', [
+  grunt.registerTask('linux-package', [
     'clean:wrap',
     'update-keys',
     'gitinfo',
     'set-custom-data',
     'release-internal',
     'bundle',
-    'electronbuilder:linux_internal',
   ]);
 
+  grunt.registerTask('linux', ['linux-package', 'electronbuilder:linux_internal']);
```

Comment:
Let's move this above the other short tasks, too.

Reply:
You mean below, right? Done